index.html

```diff
@@ -0,0 +1,1109 @@
+<!dOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>TNW</title>
+    <link rel="icon"
+        href="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/icons/apple-touch-icon-57x57.png?v=1560510772">
+
+    <link href="https://fonts.googleapis.com/css?family=Lato&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Passion+One&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.3/css/all.css"
+        integrity="sha384-UHRtZLI+pbxtHCWp1t77Bi1L4ZtiqrqD80Kn4Z8NTSRyMA2Fd33n5dQ8lWUE00s/" crossorigin="anonymous">
+</head>
+
+<body>
+    <nav>
+        <div class="nav-logo">
+            <a href="#">
+                <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
+            </a>
+        </div>
+        <div class="nav-container">
+
+            <ul class="nav-black">
+                <li><a href="#">News</a></li>
+                <li><a href="#">Events ▾</a></li>
+                <li><a href="#">Business ▾</a></li>
+                <li><a href="#">AMAs</a></li>
+                <li><a href="#">Spaces</a></li>
+                <li><a href="#">Terms &amp; Conditions</a></li>
+            </ul>
+
+            <div class="nav-white">
+                <ul class="nav-sections">
+                    <li><a href="#">Latest</a></li>
+                    <li><a href="#">Hard Fork</a></li>
+                    <li><a href="#">Plugged</a></li>
+                    <li><a href="#">Fundamentals</a></li>
+                    <li><a href="#">Work 2030</a></li>
+                </ul>
+                <ul class="nav-icons">
+                    <li>
+                        <a href="#">
+                            <i class="fas fa-search fa-flip-horizontal"></i>
+                        </a>
+                    </li>
+                    <li><a href="#">
+                            <i class="fas fa-bars"></i>
+                        </a>
+                    </li>
+                </ul>
+            </div>
+
+        </div>
+        <div class="yellow">
+
+            Watch TNW2019 talks from Nike, Google, and more →
+        </div>
+    </nav>
+
+    <main>
+        <div class="cover ">
+            <article class="cover-story-1"
+                style="background-image: url(&quot;https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2019/06/calibra-796x417.jpg&quot;);">
+
+                <div class="cover-logo">
+                    <a href="#">
+                        <img src="https://cdn0.tnwcdn.com/wp-content/themes/cyberdelia/assets/img/tnw.svg" alt="">
+                    </a>
+                </div>
+
+
```

Comment:
- [ ] Please avoid using redundant empty lines between tags and avoid using extra indentation (2 spaces suggested for each indentation). To provide this, I can suggest you to install the proper code-formatter plugins.